src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {

```

Comment:
Each type of figures should has a color, so you can create a constructor with String color, double side variables and avoid getter and setter methods. Check all types of figure